esm-admin/hooks/post_push

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# Parse image name for repo name
+tagStart=$(expr index "$IMAGE_NAME" :)
+repoName=${IMAGE_NAME:0:tagStart-1}
+
+# Tag and push image for each additional tag
+for tag in {latest,$(date +%Y%m%d%H%M%S)}; do  
+    docker tag $IMAGE_NAME ${repoName}:esm-admin-ci-${tag}
+    docker push ${repoName}:esm-admin-ci-${tag}
+done
```

Comment:
Does this get called somewhere?

Reply:
Docker hub makes a call to this script, builds the image in their system and publishes it.

I can walk you through the process if you want.